Procfile

```diff
@@ -0,0 +1 @@
+web: epicentre
```

Comment:
does this work? as it expects the binary path like `/bin/*` I believe.

Reply:
Actually I tried several version in between commits the correct should be `/bin/*` reverting to that